OpenEdXMobile/src/main/java/org/edx/mobile/module/prefs/AppFeaturesPrefs.kt

```diff
@@ -1,35 +1,35 @@
 package org.edx.mobile.module.prefs
 
 import android.content.Context
-import com.google.gson.Gson
 import dagger.hilt.android.qualifiers.ApplicationContext
 import org.edx.mobile.BuildConfig
 import org.edx.mobile.model.api.AppConfig
 import javax.inject.Inject
 import javax.inject.Singleton
 
 @Singleton
-class AppFeaturesPrefs @Inject constructor(@ApplicationContext context: Context) {
-    private val pref: PrefManager = PrefManager(context, PrefManager.Pref.APP_FEATURES)
+class AppFeaturesPrefs @Inject constructor(@ApplicationContext context: Context) :
+    PrefBaseManager(context, Pref.APP_FEATURES) {
 
-    fun isValuePropEnabled() = getAppConfig().isValuePropEnabled
+    val isValuePropEnabled get() = appConfig.isValuePropEnabled
```

Comment:
Move `isValuePropEnabled` to between `iapConfig` & `isIAPEnabled` to optimize the order of initialization.